webapp/home/templates/home/requests/menu.html

```diff
@@ -19,7 +19,7 @@ <h1> Galaxy Australia Support </h1>
         <h1 class="ga-blue"><span class="material-icons">help</span></h1>
         <div class="card-body">
           <p>
-            Need some assistance?
+            Checked <a href="/faqs.html">our FAQs</a> and still need some assistance?
             <br>
```

Comment:
I would remove this <br> as the wrapping now looks a bit weird

Reply:
Good call, done